paper-input.html

```diff
@@ -67,6 +67,10 @@
         display: block;
       }
 
+      input {
```

Comment:
I was thinking of it, but then you'd have something like this:
<img width="503" alt="screen shot 2015-10-26 at 11 23 12 am" src="https://cloud.githubusercontent.com/assets/6173664/10738291/f90a58c2-7bd3-11e5-9d94-6b0b049ee0a6.png">

is it the desired behavior?